packages/framework/src/Component/FileUpload/ImageUploadData.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Component\FileUpload;
+
+class ImageUploadData
+{
+    /**
+     * @var string[]
+     */
+    public $uploadedFiles = [];
```

Comment:
why not

Reply:
Actually it is called uploaded files to match name of FileuploadType, to make it consistent, i will not make this change since it make much more sense to me to keep it consistent with FileUploadType

Reply:
Well, FileUploadType is about files, there it is right. But this class is about images. Even other parameters of this class are named imagesToDelete and orderedImages. You should be consistent with the rest of the class and not with some not related one.

Reply:
But this change would require change of whole `FileUploadType`. because `ImageUploadData->uploadedFiles` is mapped into FileUploadType. So if I would try to rename this to uploadedImages, there would need to be `uploadedImages` Then i would need to refactor every single entity that uses `FileUploadType` and change their property from `uploadedFiles` to `uploadedImages` which would not be right.

Reply:
It's kinda weird, but I would use `$uploadedFiles` for the moment. @MattCzerner is right that it would be more complicated to make it consistent across the application.

I think that the name `$uploadedFiles` is acceptable for the time being.